pandas/tests/tools/test_numeric.py

```diff
@@ -172,7 +174,11 @@ def test_all_nan():
     tm.assert_series_equal(result, expected)
 
 
-@pytest.mark.parametrize("errors", [None, "ignore", "raise", "coerce"])
+@pytest.fixture(params=[None, "ignore", "raise", "coerce"])
```

Comment:
does this exist at a higher level? (maybe need to name differently)

Reply:
Not that I saw.